src/utils/email.ts

```diff
@@ -0,0 +1,34 @@
+import { SendEmailCommandInput } from "@aws-sdk/client-ses";
+
+export const generateParams = (schema: emailSchema): SendEmailCommandInput => {
+  return {
+    Destination: {
+      CcAddresses: [],
+      ToAddresses: [
+        schema.destination,
```

Comment:
Instead of commenting can we use an env flag like === "production"

Reply:
good idea